src/metabase/models/setting.clj

```diff
@@ -618,9 +618,16 @@
       (throw (Exception.
               (tru "Invalid value for string: must be either \"true\" or \"false\" (case-insensitive)."))))))
 
-(defn ^{:doc "The string representation of a type 4 random uuid"} random-uuid-str []
+(defn- random-uuid-str []
   (str (random-uuid)))
 
+;; This base allows the bundling of a number of attributes together. In some sense it is defining a subtype / mixin.
+(def ^{:doc "A random uuid value that should never change again"} uuid-nonce-base
```

Comment:
nit, you can put the docstring after the var name, no need for using meta tag

Reply:
Ah that's nice! In older versions of Clojure this did not work.